_examples/go.mod

```diff
@@ -5,14 +5,11 @@ go 1.18
 require (
 	github.com/alecthomas/assert/v2 v2.0.3
 	github.com/alecthomas/go-thrift v0.0.0-20170109061633-7914173639b2
-	github.com/alecthomas/kong v0.5.0
-	github.com/alecthomas/participle/v2 v2.0.0-alpha1
+	github.com/alecthomas/kong v0.6.1
```

Comment:
Wanted to get all the example tests passing to make sure I didn't break anything, so updated the go.mod file and the `thrift` test.